docs/commands/lint.md

```diff
@@ -2,239 +2,378 @@
 
 ## Introduction
 
-Redocly OpenAPI CLI can identify and report on problems found in OpenAPI definitions. This helps you avoid bugs and make API definitions more consistent.
+One of the most powerful features of Redocly CLI is its ability to identify problems found in OpenAPI definitions. This is achieved with the `lint` command, which helps your definitions conform to a consistent set of rules.
 
-The `lint` command reports on problems and executes preprocessors and rules. Unlike the `bundle` command, `lint` doesn't execute decorators.
+The `lint` command ensures that your definition file structure is valid (according to the OpenAPI Specification and your own custom rules, if any) and contains no errors. It also executes preprocessors and rules.
 
-:::success Tip
-To learn more about preprocessors and rules, refer to the [custom rules](../resources/custom-rules.md) page.
+:::warning Important
+Unlike the `bundle` command, `lint` doesn't execute decorators. To learn more about preprocessors and rules, see the [custom rules](../resources/custom-rules.md) page.
 :::
 
-## Usage
+## Options
 
-```bash
-openapi lint
-openapi lint <entrypoints>...
-openapi lint [--max-problems=<n>] [--config=<path>] [--format=<value>]
-openapi lint [--generate-ignore-file] [--help]
-openapi lint --version
+Always the over-achiever, `lint` has many options which you can execute alone, or combine with others in a single command:
+
+```bash Standalone
+redocly lint
+redocly lint <entrypoints>...
+redocly lint --config=<path>
+redocly lint --extends=<value>
+redocly lint --format=<value>
+redocly lint --generate-ignore-file
+redocly lint --help
+redocly lint --max-problems=<n>
+redocly lint --skip-preprocessor=<preprocessor>
+redocly lint --skip-rule=<rule>,<rule>
+redocly lint --version
 ```
```

Comment:
Is this part necessary? There is a table with options below. And then examples further down below.

Reply:
I can get rid of it.